data/mri_data.py

```diff
@@ -75,8 +76,9 @@ def __getitem__(self, i):
         fname, slice, padding_left, padding_right = self.examples[i]
         with h5py.File(fname, 'r') as data:
             kspace = data['kspace'][slice]
+            mask = np.asarray(data['mask']) if 'mask' in data else None
             target = data[self.recons_key][slice] if self.recons_key in data else None
             attrs = dict(data.attrs)
             attrs['padding_left'] = padding_left
             attrs['padding_right'] = padding_right
-            return self.transform(kspace, target, attrs, fname.name, slice)
+            return self.transform(kspace, mask, target, attrs, fname.name, slice)
```

Comment:
Can you update the models/unet/train_unet data transform to take this mask parameter in as well? Otherwise I suspect it won't run.

Reply:
Right, the U-Net's data transform needs to be updated as well.

Reply:
DataTransform in train_unet.py has been updated.